src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json

```diff
@@ -9,6 +9,7 @@
             "test_aks_create_with_ossku",
             "test_aks_nodepool_add_with_ossku",
             "test_aks_create_with_node_config",
+            "test_aks_create_with_http_proxy_config",
```

Comment:
Please remove this line from this file. This file is used to filter out test cases that cannot be executed correctly (the cases recorded here all depend on some feature which needs to be manually registered for the test sub).
```suggestion
```